docs/game_sdk/Store.md

```diff
@@ -0,0 +1,445 @@
+# Store
+
+> danger
+> The Discord Store is still in a beta period. All documentation and functionality can and will change.
+
+If your game has DLC or offers in-app purchases, this manager is for you! The Store Manager allows you to fetch a users' entitlements, as well as being notified when a user is granted an entitlement from a purchase flow for your game.
+
+## Application Test Mode
+
+With this new Store Manager comes a new fun toggle in the Discord app itself: Application Test Mode! You can find this toggle in your Discord user settings -> Appearance -> allllll the way at the bottom. If it's not there, make sure you have "Developer Mode" toggled on as well.
+
+So what's it do? Glad you asked! When enabled, Discord will prompt you to enter an application ID. This should be the application ID of your game. After doing so and exiting the settings, you'll see an orange banner across the top of your application. This means it worked!
+
+While in application test mode, you can freely make "purchases" of SKUs tied to your application. That means you can test buying your game, buying DLC, or going through an IAP flow without any credit card charges. It will also allow you to "purchase" unpublished SKUs, which are just SKUs that Discord has not yet included in the store directory.
```

Comment:
> which are just SKUs that Discord has not yet included in the store directory.

Eh it's a bit beyond that. Unpublished SKUs are never returned by our client facing endpoints so directly navigating to one wouldn't work unless in test mode.